### PR TITLE
Pods 8003

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: test/controllers/EventReportControllerSpec.scala [hunk 1]
```diff
@@ -434,15 +434,17 @@ class EventReportControllerSpec extends AsyncWordSpec with Matchers with Mockito
 
     }
 
-    "throw a 400 Bad Request Exception when eventType missing" in {
+    "throw a OK when eventType missing" in {
```
Comment (ngmjohnstone) [inline test/controllers/EventReportControllerSpec.scala:437]: OK not thrown hopefully!

**Review by ngmjohnstone — DISMISSED** [review]
One comment on test description

File: test/controllers/EventReportControllerSpec.scala (hunk 1 above)
Comment (grantrobson) [reply test/controllers/EventReportControllerSpec.scala]: fixed

**Review by ngmjohnstone — APPROVED** [review]
Approved

**Pull request merged**